cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -221,13 +222,13 @@ public GoogleCredential getComputeCredential() throws IOException {
    * @return GoogleCredentials for use with Dataproc
    * @throws IOException if there was an error reading the account key
    */
-  public GoogleCredentials getDataprocCredentials() throws IOException {
+  GoogleCredentials getDataprocCredentials() throws IOException {
     if (accountKey == null) {
       return getComputeEngineCredentials();
     }
 
     try (InputStream is = new ByteArrayInputStream(accountKey.getBytes(StandardCharsets.UTF_8))) {
-      return GoogleCredentials.fromStream(is);
+      return GoogleCredentials.fromStream(is).createScoped("https://www.googleapis.com/auth/cloud-platform");
```

Comment:
This string is used in multiple places now; worth a constant?

Reply:
Done